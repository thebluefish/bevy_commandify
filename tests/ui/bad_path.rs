use bevy_commandify::*;

#[command(name = foo::bar)]
fn foo(world: &mut World) { }

/// Test that a multi-part path fails
fn main() { }