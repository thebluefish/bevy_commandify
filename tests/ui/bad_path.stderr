error: Name must be an ident, found path
 --> tests/ui/bad_path.rs:3:18
  |
3 | #[command(name = foo::bar)]
  |                  ^^^^^^^^
